addons/web/static/src/core/notifications/notification.js

```diff
@@ -14,22 +14,7 @@ export class Notification extends Component {
     }
 
     get className() {
-        let className;
-        switch (this.props.type) {
-            case "danger":
-                className = "bg-danger";
-                break;
-            case "warning":
-                className = "bg-warning";
-                break;
-            case "success":
-                className = "bg-success";
-                break;
-            case "info":
-                className = "bg-info";
-                break;
-        }
-        return className ? `${className} ${this.props.className}` : this.props.className;
+        return this.props.type;
```

Comment:
what about the fallback on `this.props.className` ? This props seems to be ignored now.